tests/invoke_sandbox.rs

```diff
@@ -26,3 +26,25 @@ fn invoke_token() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn source_account_exists() -> Result<(), Box<dyn std::error::Error>> {
+    const WASM: &str = "target/wasm32-unknown-unknown/test-wasms/test_invoker_account_exists.wasm";
+    assert!(
+        Path::new(WASM).is_file(),
+        "file {WASM:?} missing, run 'make test-wasms' to generate .wasm files before running this test"
+    );
+
+    let tmp = TempDir::new().unwrap();
+    let ledger = tmp.child("ledger.json");
+
+    let mut cmd = Command::cargo_bin("soroban")?;
+    cmd.arg("invoke");
+    cmd.arg("--ledger-file").arg(ledger.as_os_str());
+    cmd.arg("--id").arg("1");
+    cmd.arg("--wasm").arg(WASM);
+    cmd.arg("--fn").arg("invkexists");
+    cmd.assert().success().stdout("true\n");
```

Comment:
how did this get true, didn't see the test create source account first, is this b/c sandbox has `GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF` and cli hardcodes that as default `--account` , if so, may be worthwhile to include pre-req/assumptions in test comments.

Reply:
Have a look in the `src/invoke.rs` file in this PR. This PR is introducing the automatic creation of the source account, whatever it is.

Reply:
doh! ok, got it, `invoke` has different implementations per the target(rpc vs sandbox), this account creation behavior is only done within path for sandbox target.